AWS/cloudformation-templates/s3-and-cloudfront.json

```diff
@@ -228,6 +228,10 @@
               "Action": ["s3:ListBucket"],
               "Effect": "Allow",
               "Resource": { "Fn::Join": ["", ["arn:aws:s3:::", { "Ref": "S3Bucket" }]]}
+            }, {
+              "Action": ["cloudfront:CreateInvalidation"],
+              "Effect": "Allow",
+              "Resource": { "Fn::Join": ["", ["arn:aws:s3:::", { "Ref": "S3Bucket" }]]}
```

Comment:
You need to change the resource to be the cloudfront distribution. However, since you can't use ARNs to refer to specific distributions you need to set this to "*"